RecoParticleFlow/PFClusterProducer/python/particleFlowClusterOOTECAL_cff.py

```diff
@@ -1,5 +1,7 @@
 import FWCore.ParameterSet.Config as cms
-from RecoParticleFlow.PFClusterProducer.particleFlowClusterECAL_cfi import *
+from RecoParticleFlow.PFClusterProducer.particleFlowClusterECAL_cff import *
 
 particleFlowClusterOOTECAL = particleFlowClusterECAL.clone()
+particleFlowClusterOOTECAL.energyCorrector.applyMVACorrections = True
+particleFlowClusterOOTECAL.energyCorrector.maxPtForMVAEvaluation = 90.
```

Comment:
this change is still here after recent updates. Note that particleFlowClusterOOTECAL once cloned already has SR-based correction enabled for 2017.
- The applyMVACorrections is redundant and does nothing
- the maxPtForMVAEvaluation overrides the value which is set to 300 for run2_ECAL_2017: does it make sense to have different thresholds for OOT?